src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/socialEmbed/index.js

```diff
@@ -1,18 +1,17 @@
 import path from 'ramda/src/path';
 
+const isSocialEmbed = ({ type }) => type === 'social_embed';
+const matchesSource = source => block => block.source === source;
+
 const convertSocialEmbed = (block, pageData) => {
   const blocks = path(['content', 'blocks'], pageData);
-  const socialEmbedMap = blocks.reduce((map, _block) => {
```

Comment:
I found this reduce function a little confusing so I refactored. will obviously need tested to ensure the whole function still works as expected